src/care/host_device_map.h

```diff
@@ -226,7 +226,13 @@ namespace care {
          }
 
          // copy constructor 
-         host_device_map(host_device_map const & other) noexcept = default;
+         CARE_HOST_DEVICE host_device_map(host_device_map const & other) noexcept {
```

Comment:
Can you use the member initializer format?

```
CARE_HOST_DEVICE host_device_map(host_device_map const & other) noexcept :
   m_max_size(other.m_max_size),
   m_signal(other.m_signal),
   ...
{
}
```

Reply:
Done